lib/activerecord-bitemporal/patches.rb

```diff
@@ -18,13 +18,14 @@ def klass
     module Persistence
       using Module.new {
         refine Persistence do
+          COPY_BITEMPORAL_OPTION_EXCEPT_KEYS = %i[ignore_bitemporal_callbacks]
           def copy_bitemporal_option(src, dst)
             return unless [src.class, dst.class].all? { |klass|
               # NOTE: Can't call refine method.
               # klass.bi_temporal_model?
               klass.include?(ActiveRecord::Bitemporal)
             }
-            dst.bitemporal_option_merge! src.bitemporal_option
+            dst.bitemporal_option_merge! src.bitemporal_option.except(*COPY_BITEMPORAL_OPTION_EXCEPT_KEYS)
```

Comment:
Why is only this option ignored?

Reply:
Behavior of whether or not a callbacks are executed depends on when to call `assign_nested_attributes`, so I thought it might cause bugs if `ignore_bitemporal_callbacks` option is copied to relation.

```rb
 # copied bitemporal_option to relation if call assign_attributes after set bitemporal_option
parent.bitemporal_option_merge!(ignore_bitemporal_callbacks: true)
parent.assign_attributes({ child_attributes: { ... } })
parent.save! # parent's and child's bitemporal_callbacks did not run

 # not copied bitemporal_option to relation if call assign_attributes before set bitemporal_option
parent.assign_attributes({ child_attributes: { ... } }) # copied bitemporal_option to relation
parent.bitemporal_option_merge!(ignore_bitemporal_callbacks: true)
parent.save! # child's bitemporal_callbacks are executed
```

But now I feel it would be more difficult that only `ignore_bitemporal_callbacks` option is treated specially. I removed this change.

Reply:
Yeah, you're right, this behavior is counter-intuitive and could be footgun. But this should be tackled as another issue.